src/displayapp/widgets/Counter.h

```diff
@@ -41,6 +41,7 @@ namespace Pinetime {
         int min;
         int max;
         int value;
+        const int leadingZeroCount;
```

Comment:
I think `const` must be removed as the `leadingZeroCount` may change when `SetMax()` is called (this update is not yet implemented in `SetMax()`

Reply:
The Counter would break anyway if the number of digits was changed with SetMax(), since resizing is not implemented, so I added a comment discouraging changing the number of digits. SetMax() is currently only used to set the number of days in a month.